triedb/pathdb/database.go

```diff
@@ -377,10 +377,10 @@ func (db *Database) Recover(root common.Hash, loader triestate.TrieLoader) error
 }
 
 // Recoverable returns the indicator if the specified state is recoverable.
-func (db *Database) Recoverable(root common.Hash) bool {
+func (db *Database) Recoverable(dest common.Hash) bool {
```

Comment:
`dest` is a bit of a bad name here as it implies a location. If you want to keep it a bit more contextual than `root`, perhaps call it `target`?

Reply:
Though I think root was fine :P

Reply:
Otherwise you lose the context that it's a root hash / state thing and not a block hash

Reply:
ok, revert to root